Source/Libraries/FaultData/DataOperations/ConfigurationOperation.cs

```diff
@@ -989,7 +980,7 @@ private void AddTriggerChannel(Channel powChannel, MeterDataSet meterDataSet)
                 MeasurementTypeID = powChannel.MeasurementTypeID,
                 MeasurementCharacteristicID = powChannel.MeasurementCharacteristicID,
                 PhaseID = powChannel.PhaseID,
-                Name = powChannel.Name,
+                Name = powChannel.Name + "Trigger",
```

Comment:
This creates a channel name like `T1Trigger`, right? Maybe `T1Trend` would make more sense.